docs/MockFunctionAPI.md

```diff
@@ -515,6 +515,19 @@ test('async test', async () => {
 });
 ```
 
+### `replacedProperty.replaceValue(value)`
```

Comment:
Take a look at The Jest Object page. I think there should be separate sections for mock function and replaced property reference. Something like:

**Methods**
- Mock Function
  [list ...]

- Replaced Property
  [list ...]